docs/docs/tutorials/tutorial_example/FashionMNIST/train.py

```diff
@@ -0,0 +1,74 @@
+import os
+import argparse
+
+import torch
+import torchvision
+import torch.nn as nn
+import torchvision.transforms as transforms
+
+from bigdl.orca import init_orca_context, stop_orca_context
+from bigdl.orca.learn.pytorch import Estimator
+from bigdl.orca.learn.metrics import Accuracy
+
+from model import model_creator, optimizer_creator
+
+parser = argparse.ArgumentParser(description='PyTorch Example')
+parser.add_argument('--cluster_mode', type=str, default="spark-submit",
+                    help='The cluster mode, such as local, yarn-client, yarn-cluster, spark-submit or k8s.')
+parser.add_argument('--backend', type=str, default="spark",
+                    help='The backend of PyTorch Estimator; '
+                         'bigdl, ray and spark are supported.')
+parser.add_argument('--batch_size', type=int, default=4, help='The training batch size')
+parser.add_argument('--epochs', type=int, default=2, help='The number of epochs to train for')
+parser.add_argument('--data_dir', type=str, default="/tmp/dataset", help='The path of dataset')
+parser.add_argument('--remote_dir', type=str, help='The path to load data from remote resources like HDFS or S3')
+parser.add_argument('--download', default=True, action='store_true', help='Download dataset or not')
+parser.add_argument('--extra-python-lib', type=str, default='model.py', help='Load dependency when running on yarn')
+args = parser.parse_args()
+
+if args.remote_dir is not None:
```

Comment:
For the data processing code, it will download data from hdfs to local and create a torch dataset on driver, not downloading to every executor. @jason-dai